.clang-format

```diff
@@ -0,0 +1,194 @@
+AllowShortLoopsOnASingleLine: true
+# Kt N/A. In Kotlin return type is in a trailing position.
+AlwaysBreakAfterReturnType: None
+# Kt U. Not touching.
+AlwaysBreakBeforeMultilineStrings: false
+# Kt N/A. In Kotlin type parameters are declared inline in a much more compact way.
+# Using Yes to make it easy to detect function name
+AlwaysBreakTemplateDeclarations: Yes
+# Kt https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting
+BinPackArguments: true
+# Kt https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting
+BinPackParameters: false
+BraceWrapping:
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterClass: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterControlStatement: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterEnum: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterFunction: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterNamespace: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterObjCDeclaration: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterStruct: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterUnion: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    AfterExternBlock: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting-control-flow-statements
+    BeforeCatch: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting-control-flow-statements
+    BeforeElse: false
+    # Kt https://kotlinlang.org/docs/reference/coding-conventions.html#formatting
+    IndentBraces: false
+    # Only used when AfterFunction is true
+    SplitEmptyFunction: true
+    # Only used when AfterRecord is true
+    SplitEmptyRecord: true
+    # Only used if AfterNamespace is true
+    SplitEmptyNamespace: true
+# Kt U. Break after operators
+BreakBeforeBinaryOperators: None
+# Configured by BraceWrapping
+BreakBeforeBraces: Custom
+# Kt U. Using true because it looks more like an if-expression with "then" and "else" branches marked "?" and ":"
+BreakBeforeTernaryOperators: true
+# Kt N/A. The closest is https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting AfterColon looks the most consistent
+BreakConstructorInitializers: AfterColon
+# Kt https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting
+BreakInheritanceList: AfterColon
+# Kt U. Choosing to break long strings to fit into line length
+BreakStringLiterals: true
+# Kt U. IDEA displays a vertical guide at 120. Choosing it to avoid very long lines.
+ColumnLimit: 140
```

Comment:
Can this cause clang-format to merge multiline statement to single line?

Reply:
Yes. See `AllowShort…OnASingleLine`